scripts/build_artifacts.py

```diff
@@ -0,0 +1,143 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
need this to also have code to create the whl and .tar.gz for pypi

Reply:
it does